### PR TITLE
[AIRFLOW-708] handling multiline output in SSHExecuteOperator

### PR DESCRIPTION
**Pull request by jhsenjaliya**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
https://issues.apache.org/jira/browse/AIRFLOW-708

### CODE REVIEW

File: airflow/contrib/operators/ssh_execute_operator.py [hunk 1]
```diff
@@ -133,24 +133,21 @@ def execute(self, context):
...
 
             self.sp = sp
-
-            logging.info("Output:")
-            line = ''
+            output = StringIO()
             for line in iter(sp.stdout.readline, b''):
-                line = line.decode().strip()
-                logging.info(line)
+                print >> output, line
```
Comment (jhsenjaliya) [inline airflow/contrib/operators/ssh_execute_operator.py]: I may need to decode it to utf-8 here, so user dont have to

Comment (bolkedebruin) [reply airflow/contrib/operators/ssh_execute_operator.py]: Is this python 3 compatible ?

Comment (jhsenjaliya) [reply airflow/contrib/operators/ssh_execute_operator.py]: No, cStringIO is not available in python 3, let me replace this with six, Thanks

**Comment by bolkedebruin** [comment]
Please also make sure to have a proper commit message per guidelines.

**Comment by sdikby** [comment]
@jhsenjaliya i suppose that this should be no more considered ?

**Comment by jhsenjaliya** [comment]
@sdikby, you are right since all open PR will go to master anyway.

**Comment by jhsenjaliya** [comment]
closing this PR since entire ssh operators is changing in #1999 which will cover multiline output.

**Pull request closed without merging**